shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/ConstraintTokenGeneratorTest.java

```diff
@@ -72,6 +71,6 @@ public void assertGenerateSQLTokens() {
         constraintTokenGenerator.setShardingRule(shardingRule);
         Collection<ConstraintToken> result = constraintTokenGenerator.generateSQLTokens(alterTableStatementContext);
         assertThat(result.size(), is(1));
-        assertThat(result.stream().collect(Collectors.toList()).get(0).getStartIndex(), is(TEST_START_INDEX));
+        assertThat((new LinkedList<>(result)).get(0).getStartIndex(), is(TEST_START_INDEX));
```

Comment:
Minimize the use of variables.
The life cycle of the constant defined in the class is bound to the bytecode file.
If `TEST_START_INDEX` is only used in methods, use local variables. The life cycle of the variable is bound to the method.